addons/edi/tests/common.py

```diff
@@ -188,3 +188,17 @@ def assertRaisesIssue(self, entity, exception=UserError):
 
         # Delete the raised issue
         new_issue_ids.unlink()
+
+    def _sim_edi_cron_execution(self, document_type):
```

Comment:
This function can be used for other tests not simulating crons, so I would suggest to rename it to something more generic, and we probably do not need references to crons.

Reply:
something like this?
```python
def _execute_edi_for(self, document_type):
   ...

# Usage
self._execute_edi_for(self.document_type_abc)
```

Reply:
i'm never too hot on naming things, especially generic functions

Reply:
That sounds good to me. I'm not great with names either :)